### PR TITLE
KOTL and OD buffs

### PR DESCRIPTION
**Pull request by POCKETMONEY99**
Rescaled KOTL illuminate to be more consistent at OAA levels. 
Rescaled KOTL blinding light cooldown to buff his early game.
Reduced Cooldown on Will-o-Wisp as this is his only "good spell" 

Reduced Mana Cost of Arcane Orb to make OD's game easier at all stages. 
Increased the damage on Arcane Orb slightly to compensate for it being magical damage.
Rescaled cooldown of Astral imprisonment to buff OD's early game farming and rune contest. 
Reduced cooldown of Sanity's Esciple. (160 second cooldown is too long cooldown for any spell.)

### CODE REVIEW

File: game/scripts/npc/abilities/obsidian_destroyer_arcane_orb_oaa.txt [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
     // Cost
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityManaCost"                                     "110 140 170 200 260 360"
+    "AbilityManaCost"                                     "100 120 140 160 200 240"
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/obsidian_destroyer_arcane_orb_oaa.txt]: Don't buff mana costs. It doesn't change much.

File: game/scripts/npc/abilities/obsidian_destroyer_arcane_orb_oaa.txt [hunk 2]
```diff
@@ -32,7 +32,7 @@
       "01"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "mana_pool_damage_pct"                            "6 7 8 9 10 11"
+        "mana_pool_damage_pct"                            "7 8 9 10 11 12"
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/obsidian_destroyer_arcane_orb_oaa.txt]: //OAA

File: game/scripts/npc/abilities/pugna_life_drain.txt [hunk 3]
```diff
@@ -44,7 +44,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "health_drain"                                    "150 225 300 500 750"
+        "health_drain"                                    "150 250 350 650 950" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/pugna_life_drain.txt]: With decrepify and spell amp items you think this is a good idea? Meta right now are spell-casters anyway.

File: game/scripts/npc/abilities/obsidian_destroyer_arcane_orb_oaa.txt [hunk 4]
```diff
@@ -22,7 +22,7 @@
 
     // Cost
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityManaCost"                                     "110 140 170 200 260 360"
+    "AbilityManaCost"                                     "110 140 170 200 230 260"
```
Comment (lovesmuffins) [inline game/scripts/npc/abilities/obsidian_destroyer_arcane_orb_oaa.txt]: After playing OD these are the changes that are needed. A buff to his early game Q manacost and a nerf to his lategame Q. OD has big mana issues early game but 0 mana issues late game (makes E really useless)

I suggest that you chang themanacost to 90/110/130/150/300/500

File: game/scripts/npc/abilities/obsidian_destroyer_arcane_orb_oaa.txt [hunk 5]
```diff
@@ -32,7 +32,7 @@
       "01"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "mana_pool_damage_pct"                            "6 7 8 9 10 11"
+        "mana_pool_damage_pct"                            "7 8 9 10 11 12" //OAA
```
Comment (lovesmuffins) [inline game/scripts/npc/abilities/obsidian_destroyer_arcane_orb_oaa.txt]: I don't think this will change anything. Maybe increase int steal by 1 at all levels instead or make the duration scale a bit.

File: game/scripts/npc/abilities/obsidian_destroyer_equilibrium.txt [hunk 6]
```diff
@@ -41,7 +41,7 @@
       "04"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "duration"                                        "7"
+        "duration"                                        "7 8 9 10 11 12" //OAA
```
Comment (lovesmuffins) [inline game/scripts/npc/abilities/obsidian_destroyer_equilibrium.txt]: mana steal needs a buff early game but doesn't need to scale as much. Leave duration at 7.
Mana steal should be 90/110/130/150/160/170 or something like a constant 125 at all levels would be fine.

File: game/scripts/npc/heroes/obsidian_destroyer.txt [hunk 7]
```diff
@@ -8,5 +8,9 @@
     // Abilities
     //-------------------------------------------------------------------------------------------------------------
     "Ability1"    "obsidian_destroyer_arcane_orb_oaa" // replaces obsidian_destroyer_arcane_orb
+
+    //Attributes
+    //-------------------------------------------------------------------------------------------------------------
+    "AttributeAgilityGain"		"2.400000"
```
Comment (lovesmuffins) [inline game/scripts/npc/heroes/obsidian_destroyer.txt]: I would remove this as this is an unnecessary change.

File: game/scripts/npc/abilities/obsidian_destroyer_arcane_orb_oaa.txt (hunk 5 above)
Comment (POCKETMONEY99) [reply game/scripts/npc/abilities/obsidian_destroyer_arcane_orb_oaa.txt]: This is because the spell does magic damage. The percentage remained the same despite it being changed to magic damage

File: game/scripts/npc/heroes/obsidian_destroyer.txt (hunk 7 above)
Comment (POCKETMONEY99) [reply game/scripts/npc/heroes/obsidian_destroyer.txt]: Attack speed hero in OAA which isn't an agi hero. This is why

Comment (lovesmuffins) [reply game/scripts/npc/heroes/obsidian_destroyer.txt]: I'd rather you just reduce his BAT by 1 than. .4 agility x 50 is 20 extra attack speed at max level. That means almost nothing tbh
